ckanext/cfpb_extrafields/digutils.py

```diff
@@ -0,0 +1,94 @@
+from openpyxl import load_workbook
+try:
+    from ckan.plugins.toolkit import Invalid
+except ImportError: # pragma: no cover
+    #If the custom exception can't be imported, use a more generic exception
+    Invalid = Exception
+
+from ckanext.cfpb_extrafields import validators as v
+
+# Helper functions to convert values of sheets
+def strfy(val):
+    if isinstance(val, basestring):
+        return val
+    else: # pragma: no cover
+        return ""
+
+# FIELDS values can be functions that take in a worksheet and provide a value
+# These factories return functions to use there.
+# For instance, pra_exclusion is in fields D38 and D39.
+# Therefore, we want the FIELDS value to be a function that takes in a worksheet and returns
+# those cells added together.
+# Instead of making this function manually with `dig_func = lambda ws: ws["D38"]+ws["D39"]`,
+# we can say `dig_func = concat(["D38", "D39"])` and it will make the function and return it to us.
+def concat(fields):
+    def concat_fields(ws):
+        result = ""
+        for field in fields:
+            result += strfy(ws[field].value)
+        return result
+    return concat_fields
+
+def date(cell):
+    def get_date(ws):
+        val = ws[cell].value
+        if hasattr(val, "strftime"):
+            val = val.strftime("%Y-%m-%d")
+        _ = v.reasonable_date_validator(val)#Make sure it's a valid date, but return the string.
```

Comment:
Can we format this comment like the others with proper spacing?  Picky I know, but I like clean comments :)